src/alert/alert.spec.ts

```diff
@@ -99,6 +99,29 @@ describe('NgbDismissibleAlert', () => {
            });
      })));
 
+  it('should auto close on timeout specified', fakeAsync(inject([TestComponentBuilder], (tcb) => {
+       const html = '<template ngbAlert [dismissOnTimeout]="1000">Hello, {{name}}!</template>';
+       tcb.overrideTemplate(TestComponent, html).createAsync(TestComponent).then((fixture) => {
+         fixture.detectChanges();
+         const alertEl = getAlertElement(fixture.nativeElement);
+         fixture.detectChanges();
+         expect(alertEl.getAttribute('role')).toEqual('alert');
+         expect(getCloseButton(alertEl)).toBeTruthy();
+
+         tick(1000);
```

Comment:
Could we use a lower number (say `800`) to make it clear that we are testing what is happening before timeout?